### DISCUSSION TITLE
Add capability 'Eventmanager show own events'

### DISCUSSION THREAD

**Discussion by JanineKall** [liedekef/events-made-easy#463] [Ideas]
Hi. I manage a website with several active EME-eventmanagers. Until recently, they could only see their own events. But something changed and all events are shown to all eventmanagers. As we have a lot of published events but also a lot of events in concept, this is confusing and it takes time for them to find their own specific events. Is it possible to add a capability 'Eventmanager show own events'?

**Comment by liedekef** [comment]
Go in the EME access settings, there's now an option that controls that ("Limit event listing"):
![image](https://github.com/liedekef/events-made-easy/assets/75667/00f90377-aac8-4b04-9c00-4704c3285908)

**Reply by JanineKall** [reply]
Thanks! I overlooked.

**Reply by JanineKall** [reply]
Hi Franky,
somehow this option limits to future events, so the eventmanagers cannot duplicate a past event and use all conditions for a new event or do some administration afterwards. What am I overlooking?

**Reply by liedekef** [reply]
They should be able to filter on "Past events" or "All events" just fine ...

**Reply by JanineKall** [reply]
Yes, they should, but they can't. I captured two videos: one as an [eventmanager with restricted acces](https://youtu.be/J3ArkhdIFi0) to their own events, and one [as an admin ](https://youtu.be/ez6qeH4nSOg)to prove it. 
It's weird. The website also uses PublishPress capabilities. Should I add a capability there based on the new feature 'Limit events listing'?

I also added a screenshot of the EME capabilities setting.
![image](https://github.com/liedekef/events-made-easy/assets/129589814/023ae07d-3da2-441a-82f4-c338326eb9b3)

**Reply by liedekef** [reply]
your video is not showing all screen popups, so I can't say anything based on those. I just checked the code and in no way is there a limit set on the filter for past/future/all events.
Feel free to give me a temp account that shows the problem, so I can check on your side a little bit.

**Reply by JanineKall** [reply]
I sent you a password link

**Reply by liedekef** [reply]
If I am to see the issue at your end, you need to edit my account to limit the rights like for a problematic user.

**Reply by JanineKall** [reply]
Done

**Reply by liedekef** [reply]
ok, a typo discovered. This fixes it:
https://github.com/liedekef/events-made-easy/commit/0943ae5f1a78b1f7a62220360eb9958ce798063a

**Reply by JanineKall** [reply]
I see! Should I download EME and install it over the current plugin?

**Reply by liedekef** [reply]
Absolutely not ... just apply the one-line change manually. If unsure, give me full admin again and I'll fix it

**Reply by liedekef** [reply]
Did you got it to work?

**Reply by JanineKall** [reply]
No, not yet. Most themes offer php-editing, but the X-theme does not and I don't have access to the server.

**Reply by JanineKall** [reply]
I really want to learn to do everything myself, but now I am a little stuck. So I gave you full admin again.

**Reply by liedekef** [reply]
it's not a theme, it's a plugin. And the plugin editor is activated on wordpress level, not theme level.

**Reply by liedekef** [reply]
The plugin editor is deactivated, so that needs to be changed first :-)

Edit: I installed a file manager plugin, edited the file and removed the file manager again. It should be ok now.
Next to that: activate the automatic update of plugins and themes as much as possible (all of them) and check/install a security plugin if not yet done.

**Reply by JanineKall** [reply]
Thanks, I meant the plugin editor but I thought it came with the theme. Thanks for explaining. 
I tested it and it works.
